src/index.html

```diff
@@ -9,10 +9,51 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+      <li class="stars__list-item"></li>
```

Comment:
```suggestion
      <li class="stars__star"></li>
```
Probably its suits better because immediately by the class you understand that this is star of the block of stars